src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  &__card {
+    margin: 0 $cardMarginGorizotnal $cardMarginVertical 0;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
```

Comment:
what about the bottom margins for the last 4 cards(last line)? Do we have it? If yes, why do we have it?)

Reply:
I'm sorry, but I don't understand what the problem is and what needs to be fixed :(